src/ert/config/queue_config.py

```diff
@@ -98,7 +98,7 @@ def driver_options(self) -> dict[str, Any]:
 
 @pydantic.dataclasses.dataclass
 class LsfQueueOptions(QueueOptions):
-    name: Literal[QueueSystem.LSF] = QueueSystem.LSF
+    name: Literal[QueueSystem.LSF, "lsf", "LSF"] = "lsf"
```

Comment:
`QueueSystem.LSF` is `LSF` already or ?

Reply:
Yes, could probably either remove that, do a `lower` or even drop the enum. Not sure what would be the best approach.